src/radical/pilot/agent/scheduler/continuous.py

```diff
@@ -420,7 +429,8 @@ def _alloc_nompi(self, cud):
                    total_gpus, self._lrms_gpus_per_node)
```

Comment:
Do we have a problem with the indentation of the previous line (428)? (it may be an issue with github diff mode)

Reply:
I don't think there is any indentation issue here. Checked in code.